### PR TITLE
[EP-2214] Implement custom cypress docker image

### PR DESCRIPTION
**Pull request by jlison**
Implement custom cypress docker image.
* Google Chrome binaries: v.69
* Node: v.10
* Cypress: v3.7.0

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# https://circleci.com/orbs/registry/orb/circleci/docker
+version: 2.1
+orbs:
+    docker: circleci/docker@0.5.18
+workflows:
+    build_and_publish_docker_image:
+        jobs:
+            -   docker/publish:
```
Comment (deluxetom) [inline .circleci/config.yml:8]: you should probably have 1 build with the `latest` tag too

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# https://circleci.com/orbs/registry/orb/circleci/docker
```
Comment (cryptiklemur) [inline .circleci/config.yml:1]: Should move this to buildkite, no?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,23 @@
+FROM cypress/browsers:chrome69
```
Comment (cryptiklemur) [inline Dockerfile]:
```suggestion
ARG BROWSER="chrome69"
FROM cypress/browsers:$BROWSER
```

Comment (cryptiklemur) [reply Dockerfile]: Would allow us to build an image for multiple browsers

File: README.md [hunk 4]
```diff
@@ -1,2 +1,8 @@
 # cypress-eplay
 Docker image for cypress tests
+
+A specialized box containing:
+
+* Google Chrome binaries: v.69
+* Node: v.10
```
Comment (cryptiklemur) [inline README.md]: I hope?

```suggestion
* Node: v10
```

File: .circleci/config.yml (hunk 2 above)
Comment (deluxetom) [reply .circleci/config.yml:1]: circleci is very easy for this and thats what I use for docker-symfony, we could move it later :)

File: README.md (hunk 4 above)
Comment (jlison) [reply README.md]: 🤣 Yep, yep

File: Dockerfile (hunk 3 above)
Comment (jlison) [reply Dockerfile]: Yeah, I love the suggestion 😄

File: .circleci/config.yml (hunk 2 above)
Comment (cryptiklemur) [reply .circleci/config.yml:1]: Just trying to move us off circle quicker. Less time we are on two CI systems, the better

**Review by cryptiklemur — APPROVED** [review]

**Pull request merged**